app/src/main/python/gogdl/api.py

```
@@ -0,0 +1,118 @@
+import logging
+import time
+import requests
+import json
+from multiprocessing import cpu_count
+from gogdl.dl import dl_utils
+from gogdl import constants
+import gogdl.constants as constants
```

Ruff (0.12.2)

Redefinition of unused `constants` from line 7

Remove definition: `constants`

(F811)